mix.exs

```diff
@@ -42,7 +48,7 @@ defmodule Move.MixProject do
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:number, "~> 1.0"}
+      {:number, "~> 1.0"},
```

Comment:
I think `mix format` removes the trailing comma. So, I would prefer to not have it.

Reply:
Oh, what a shame. It makes it harder to add a new dep and harder to review pull requests adding a dep.